tensorflow/BUILD

```diff
@@ -63,9 +63,16 @@ package_group(
     packages = ["//tensorflow/..."],
 )
 
-sh_binary(
+filegroup(
     name = "swig",
-    srcs = ["tools/swig/swig.sh"],
+    srcs = select({
+        "//tensorflow:windows" : [
+	    "tools/swig/swig.bat"
+        ],
+        "//conditions:default": [
+            "tools/swig/swig.sh"
+        ],
+    }),
```

Comment:
I believe you are right. We can just run a shell command instead of using swig as an executable.
But these is currently a bug in Bazel, https://github.com/bazelbuild/bazel/issues/1926
I have to move args to `command` argument temporarily. Please see the update.